network/vpack/parse.go

```diff
@@ -0,0 +1,292 @@
+// Code generated by gen.go; DO NOT EDIT.
+
+package vpack
+
+import (
+	"fmt"
+)
+
+func parseVote(data []byte, c compressWriter) error {
+	p := newParser(data)
+
+	// Fixed size struct with 3 fields
+	cnt, err := p.readFixMap()
```

Comment:
The fields are `omitempty`, but here we assume they will never be omitted. I wonder if we should remove the `omitempty`. I think it's true that they can never be empty.  I'm almost like a msgpack `failempty` tag.